src/main/scala/org/apache/mesos/chronos/scheduler/api/JobManagementResource.scala

```diff
@@ -151,7 +151,8 @@ class JobManagementResource @Inject()(val jobScheduler: JobScheduler,
       require(jobGraph.lookupVertex(jobName).isDefined, "Job '%s' not found".format(jobName))
       val job = jobGraph.getJobForName(jobName).get
       log.info("Manually triggering job:" + jobName)
-      jobScheduler.taskManager.enqueue(TaskUtils.getTaskId(job, DateTime.now(DateTimeZone.UTC), 0), job.highPriority)
+      jobScheduler.taskManager.enqueue(TaskUtils.getTaskId(job, DateTime.now(DateTimeZone.UTC), 0, Option(arguments).filter(_.trim.nonEmpty))
```

Comment:
If it's a PITA and exists in other parts of the codebase as well, maybe
just file a GH issue for it (because it's a problem in general) so as to
not block this PR

On Wed, Aug 24, 2016 at 8:27 AM, Ken Sipe <notifications@github.com> wrote:

> In src/main/scala/org/apache/mesos/chronos/scheduler/api/
> JobManagementResource.scala
> <https://github.com/mesos/chronos/pull/717#discussion_r76046360>:
>
> > @@ -151,7 +151,8 @@ class JobManagementResource @Inject()(val jobScheduler: JobScheduler,
> >        require(jobGraph.lookupVertex(jobName).isDefined, "Job '%s' not found".format(jobName))
> >        val job = jobGraph.getJobForName(jobName).get
> >        log.info("Manually triggering job:" + jobName)
> > -      jobScheduler.taskManager.enqueue(TaskUtils.getTaskId(job, DateTime.now(DateTimeZone.UTC), 0), job.highPriority)
> > +      jobScheduler.taskManager.enqueue(TaskUtils.getTaskId(job, DateTime.now(DateTimeZone.UTC), 0, Option(arguments).filter(_.trim.nonEmpty))
>
> its a very good point. It seems that is missed in other parts of the code
> as well. This was suppose to be just a bug fix. That and the fact we are
> looking to move the whole thing to metronome I'm not sure if makes sense to
> spend too much time on it but I will have a look. At least to get rid of
> the semi-colon.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesos/chronos/pull/717/files/40a7c0b42f1da0b3c60ae7989cf1d44e18d3cb3c#r76046360>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLIlU0O1VzssA_dXMU0SEy9YdkgB2ks5qjDiggaJpZM4Jrgou>
> .
>